django_project/ford3/views/views.py

```diff
@@ -39,7 +39,8 @@ def show_campus(request, provider_id, campus_id):
         id=campus_id)
     context = {
         'campus': campus,
-        'provider': campus.provider
+        'provider': campus.provider,
+        'provider_logo': campus.provider.logo_url
```

Comment:
Same comment with above. You can access provider by `campus.provider` in the template. I don't think adding more context has any benefit.